packages/foundation/src/layout/stack.css

```diff
@@ -0,0 +1,11 @@
+.hap-stack {
```

Comment:
Wieso wird hier justify und wrap nicht unterstützt?

Reply:
Zugegeben, man muss eine feste Höhe für den Container setzen. Aber dann funktioniert es genau so, wie für eine row.
Beispiel für wrap: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Mastering_wrapping_of_flex_items

Reply:
After discussion: maybe we'll add it later when needed.